tests/factories/user.py

```diff
@@ -9,3 +9,4 @@ class Meta:
 
     username = factory.Sequence(lambda n: f"user-{n}")
     email = factory.Sequence(lambda n: f"user-{n}@example.com")
+    fullname = factory.SelfAttribute("username")
```

Comment:
I think it's worth making the generated `username` and `fullname` distinct so there's no risk of a test mistakenly passing if one is mistakenly used where the other is expected. Maybe something like:

```suggestion
    fullname = factory.LazyAttribute(lambda obj: obj.username.replace("-", " ").title())
```

I think that "looks" more like a full name (`User 43` vs. `user-43` as the `username`).

Reply:
That's an excellent suggestion, I'll check that it still works with all the tests.